tests/src/test/java/alluxio/shell/command/StatCommandTest.java

```diff
@@ -37,15 +37,15 @@ public void fileInfoNotExist() throws IOException {
   public void fileInfoWildCard() throws IOException, AlluxioException {
```

Comment:
This should probably be named `statFileWildCard` or something similar.

Are there any tests you could write for stat'ing directories?

Also, does directories with with wildcard?